posthog/heatmaps/test/test_heatmaps_api.py

```diff
@@ -306,12 +329,12 @@ def _create_heatmap_event(
                 "team_id": team_id,
                 "distinct_id": distinct_id,
                 "timestamp": format_clickhouse_timestamp(date_from),
-                "x": 10 / 16,
-                "y": y / 16,
+                "x": round(x / 16),
+                "y": round(y / 16),
                 "scale_factor": 16,
                 # this adjustment is done at ingestion
-                "viewport_width": math.ceil(viewport_width / 16),
-                "viewport_height": math.ceil(viewport_height / 16),
```

Comment:
but the test input data doesn't have the same shape as the real ingestion will generate